src/ontology/edits/oeo-model.omn

```diff
@@ -526,6 +526,26 @@ ObjectProperty: OEO_00140164
         OEO_00000274
     
     
+ObjectProperty: OEO_00390023
+
+    Annotations: 
+        OEO_00020426 "Added
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1901
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1944",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A relation between a data item and an aggregation type that indicates the method of data aggregation applied to the data set.",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "A relation between a data item or data set and an aggregation type that indicates the method of data aggregation applied to the data set.",
```
Add data set to the definition, too? For users that are not too familiar with the oeo hierarchy this might be helpful, although data item includes data set technically.